src/Control/Director.php

```diff
@@ -932,10 +932,11 @@ public static function fileExists($file)
      */
     public static function absoluteBaseURL()
     {
-        return self::absoluteURL(
+        $baseURL = self::absoluteURL(
             self::baseURL(),
             self::ROOT
         );
+        return rtrim($baseURL, '/') . '/';
```

Comment:
Should it be adding a trailing slash even when the config says not to add them?

Reply:
The base URL should always needs a trailing slash, otherwise the graphql calls in the CMS don't work. I have no idea about graphql, so I can't fix it on that side.

Reply:
I'm going through and resolving this now - the problem seems to be simply that in the javascript in admin we're expecting there to always be a trailing slash. So I'll update the JS and then probably add a new commit to this PR to tidy up these comments to match.

Reply:
awesome, thanks!